### ISSUE TITLE
`Invalid text boundingbox error` in `TimeSeries`

### ISSUE THREAD

**Issue by fatteneder** [JuliaPlots/Makie.jl#1669]
Hi.

I wanted to contribute a `timeseries` example to #1639.
But when I try to follow the docs for `timeseries` the following happens

```julia
julia> signal = Observable(1.0)
Observable{Float64} with 0 listeners. Value:
1.0

julia> scene = timeseries(signal)
┌ Warning: No strict ticks found
└ @ PlotUtils ~/.julia/packages/PlotUtils/rTTui/src/ticks.jl:333
┌ Warning: No strict ticks found
└ @ PlotUtils ~/.julia/packages/PlotUtils/rTTui/src/ticks.jl:333
┌ Warning: No strict ticks found
└ @ PlotUtils ~/.julia/packages/PlotUtils/rTTui/src/ticks.jl:333
┌ Warning: No strict ticks found
└ @ PlotUtils ~/.julia/packages/PlotUtils/rTTui/src/ticks.jl:333
┌ Warning: No strict ticks found
└ @ PlotUtils ~/.julia/packages/PlotUtils/rTTui/src/ticks.jl:333
┌ Warning: No strict ticks found
└ @ PlotUtils ~/.julia/packages/PlotUtils/rTTui/src/ticks.jl:333
ERROR: Invalid text boundingbox
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:33
  [2] boundingbox(glyphcollection::Makie.GlyphCollection, position::Point{3, Float32}, rotation::Quaternionf)
 ...
```

A similar error message was mentioned as a side note in #1490.
However, when trying their MWE now gives a different error (related to invalid range steps for `log10` scaling). I will open a separate issue for this.

**Comment by fatteneder** [comment]
The errors for `lines` and `timeseries` are of different origin.
The former is discussed in #1670 and a PR is in progress.

For the `timeseries` plot however, I am struggling a little.
I got it to work somehow by 
- resetting the linewidths in `src/basic_recipes/axis.jl:draw-axis3d` to a finite number, because they were `NaN`s,
- manually overwriting the positions in `src/layouting/boundingbox.jl`, again to avoid `NaNs`.

With those changes I now get an empty plot window. Interestingly, it also seems to use a 3D axis now which I think is wrong.

A MWE for how it should work is given in #736

**Comment by fatteneder** [comment]
#1770 will fix this

**Comment by fatteneder** [comment]
#1770 is available with `v0.17.1` and the error is fixed.
